tensorboard/webapp/reducer_config.ts

```diff
@@ -12,32 +12,32 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-import {InjectionToken} from '@angular/core';
+import {InjectionToken, isDevMode} from '@angular/core';
 import {
   Action,
   ActionReducer,
   ActionReducerMap,
   MetaReducer,
 } from '@ngrx/store';
 
+import {storeLogger} from 'ngrx-store-logger';
+
 export interface State {}
 
-// console.log all actions
-export function logger(reducer: ActionReducer<any>): ActionReducer<any> {
-  return (state, action) => {
-    const result = reducer(state, action);
-    console.groupCollapsed(action.type);
-    console.log('prev state', state);
-    console.log('action', action);
-    console.log('next state', result);
-    console.groupEnd();
-
-    return result;
-  };
+export function logger(reducer: ActionReducer<State>): ActionReducer<State> {
+  // We must call isDevMode() within this function, not at file load time,
+  // because of load ordering issues vs. the config/bootstrap setup.
+
+  // TODO(stephanwlee, davidsoergel): Why is this called twice?
```

Comment:
I'm curious as well.  It seems ngrx calls metareducers once when creating Reducers, and again in 'updateReducers', called when it actually adds features/reducers to the store.

Maybe we can just do this console.log outside of this function, in the module itself?